addons/info/src/index.js

```diff
@@ -83,12 +67,25 @@ export function addInfo(storyFn, context, info, _options) {
   );
 }
 
-export const withInfo = (info, _options) =>
-  addonCompose((storyFn, context) => addInfo(storyFn, context, info, _options));
+export const withInfo = textOrOptions => {
```

Comment:
IMO, it would be nice to provide here such an option as well:
```js
withInfo('text info', { options })
```

There are two reasons for that:

1.1. We might want to use different `text info` with the same other options:

```js
.add('Button', withInfo('Default Button', widgetInfoOptions)(() => <Button />))
```

1.2. We might want to use multiline template string for ``` `text info` ``` and such syntax would be less verbose

2. It's relevant to the previous behavior of `.addWithInfo` when we could pass (or not) `text info` as a separate argument